tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -559,9 +559,12 @@ def test_related_field_has_invalid_related_name(self):
             'contains_%s_whitespace' % whitespace,
             'ends_with_with_illegal_non_alphanumeric_%s' % illegal_non_alphanumeric,
             'ends_with_whitespace_%s' % whitespace,
-            # Python's keyword
-            'with',
+            'with', # Python's keyword
+            'related_name\n',
         ]
+        from django.utils import six
+        if six.PY3:
+            invalid_related_names.append(chr(65292))  # ord("，")==65292
```

Comment:
`'，'` is invalid python2 .
It must be unicode string `u'，'` which is not supported by related_name.
It doesn't need test this in python2.
So only test it in python3.

I think use the string directly is better than `chr()`. I will update it.